esx_service/cli/vmdkops_admin.py

```diff
@@ -172,83 +175,186 @@ def commands():
                 }
             }
         },
-        'role': {
+        #
+        # tenant {create, rm , ls} - manipulates tenants
+        # tenant vm {add, rm, ls}  - manipulates VMs for a tenant
+        # tenant access {add, set, rm, ls} - manipulates datastore access right for a tenant
+        #
+        'tenant': {
             'help': 'Administer and monitor volume access control',
             'cmds': {
                 'create': {
-                    'func': role_create,
-                    'help': 'Create a new role',
+                    'func': tenant_create,
+                    'help': 'Create a new tenant',
                     'args': {
                         '--name': {
-                            'help': 'The name of the role',
+                            'help': 'The name of the tenant',
                             'required': True
                         },
-                        '--matches-vm': {
-                            'help':
-                            'Apply this role to VMS with names matching Glob',
-                            'metavar': 'Glob1,Glob2,...',
-                            'required': True,
+                        # a shortcut allowing to add VMs on Tenant Create
+                        '--vm-list': {
+                            'help': 'A list of VM names to place in this Tenant',
+                            'metavar': 'vm1, vm2, ...',
                             'type': comma_seperated_string
-                        },
-                        '--rights': {
-                            'help': 'Permissions granted to matching VMs',
-                            'required': True,
-                            'choices': ['create', 'delete', 'mount'],
-                            'metavar': 'create,delete,mount'
-                        },
-                        '--volume-maxsize': {
-                            'help':
-                            'Maximum size of the volume that can be created',
-                            'required': True,
-                            'metavar': 'Num{MB,GB,TB} - e.g. 2TB'
                         }
                     }
                 },
                 'rm': {
-                    'func': role_rm,
-                    'help': 'Delete a role',
+                    'func': tenant_rm,
+                    'help': 'Delete a tenant',
                     'args': {
                         'name': {
-                            'help': 'The name of the role'
-                        }
+                            'help': 'The name of the tenant',
+                      },
+                      '--remove-volumes': {
+                        'help': 'BE CAREFUL -  Removes this tenant volumes when removing a tenant'
+                      }
```

Comment:
Or something like "WARNING: - Removes this tenant's volumes from all datastores when removing a tenant